docs/cookbook/wasm.md

```diff
@@ -1,4 +1,9 @@
-# WASM Example
+# WASM Example (JavaScript)
```

Comment:
I think WASM is well known in its own right, no need to mention that it works with js IMO.